README.md

```diff
@@ -74,10 +73,96 @@ supervisorctl status cosmian_kms
 ## Server parameters
 
 If a configuration file is provided, parameters are set following this order:
-- conf file (env variable `KMS_SERVER_CONF` set by default to `/etc/cosmian_kms/conf.toml`)
+- conf file (env variable `KMS_SERVER_CONF` set by default to `/etc/cosmian_kms/server.toml`)
 - default (set on struct)
 
 otherwise the parameters are set following this order:
 - args in command line
 - env var
 - default (set on struct)
+
+## Use the KMS inside a Cosmian VM on SEV/TDX
+
+See [this README](https://github.com/Cosmian/cosmian_vm) for more details about Cosmian VM. 
+
+To deploy the KMS inside a Cosmian VM, follow these steps:
+
+```console
+$ # Copy from resources/supervisor/cosmian_kms.ini
+$ sudo vi /etc/supervisord.d/cosmian_kms.ini
+$ # Copy the kms server binary
+$ sudo cp somelocation/cosmian_kms_server /usr/sbin/cosmian_kms
+$ # Create a conf file for the KMS (from resources/server.toml)
+$ sudo vi /etc/cosmian_kms/server.toml
+$ sudo supervisorctl reload 
+$ sudo supervisorctl start cosmian_kms
+$ # Check logs
+$ sudo tail -f /var/log/cosmian_kms.err.log
+$ sudo tail -f /var/log/cosmian_kms.out.log
+```
+
+Now you can interact with your KMS through the KMS CLI. 
+
+You can also interact with the Cosmiam VM Agent through its own CLI as follow:
+
+```console
+$ # From your own host
+$ # Snapshot the VM (it could take a while)
+$ ./cosmian_vm --url https://domain.name:port snapshot 
+$ # Sometimes, verify it
+$ ./cosmian_vm --url https://domain.name:port verify --snapshot ./cosmian_vm.snapshot
+Reading the snapshot...
+Fetching the collaterals...
+[ OK ] Verifying TPM attestation
+[ OK ] Verifying TEE attestation
+```
+
+You can also provide the configuration file of the KMS through the Cosmian VM Agent and let it start the KMS.
+
+1. Adapt the `/etc/supervisord.d/cosmian_kms.ini` by adding: `environment=COSMIAN_KMS_CONF=/mnt/cosmian_vm/data/app.conf`
+2. Add the following lines in `/etc/cosmian_vm/agent.toml`
+```toml
+[app]
+service_type = "supervisor"
+service_app_name = "cosmian_kms"
+decrypted_folder = "/mnt/cosmian_vm/data"
+encrypted_secret_app_conf = "/etc/cosmian_kms/server.toml"
+```
+3. Provider the configuration (where `conf.toml` is the configuration file of the KMS):
+```console
+$ ./cosmian_vm --url https://domain.name:port app init  -c conf.toml 
+Proceeding the init of the deployed app...
+Save the key: `378f1f1b3b5cc92ed576edba265cc91de6872d61c00b0e01dba6d0ea80520820`
+The app has been configurated
+```
+4. Save the prompted key for further use
+5. In case of reboot, u will need to restgart the KMS manually by sending the configuration decryption key:
```

Comment:
```suggestion
5. In case of reboot, you will need to restart the KMS manually by sending the configuration decryption key (the key saved at step 4):
```